Cargo.toml

```diff
@@ -8,4 +8,5 @@ authors = [
 edition = "2018"
 
 [dependencies]
-log = "0.4.11"
\ No newline at end of file
+log = "0.4.11"
+pretty_env_logger = "0.4.0"
```

Comment:
`pretty_env_logger` should be in `build_dependencies` so that it is only built when testing.